script/test-in-container.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+testcase=$1
+set -e pipefail
```

Comment:
I don't prefer having this in the shebang because that's just a comment after all. If you invoke the script like `bash test-in-container.sh` the `-e` in that comment has no effect.

Reply:
no, the shebang is *not* a comment, weird convention maybe, yes, but it's not a comment. That you can override settings by calling it explicitly is a feature :)

Reply:
From `bash`s perspective is *is* a comment. That the script behaves differently when invoking the interpreter manually is likely nothing one is usually expects so I consider it bad or maybe even a bug.